pkg/api/frontend_logging.go

```diff
@@ -28,14 +66,121 @@ type frontendSentryEvent struct {
 	Exception *frontendSentryException `json:"exception,omitempty"`
 }
 
+func guessSourceMapLocation(sourceUrl string) (*sourceMapLocation, error) {
+	u, err := url.Parse(sourceUrl)
+	if err != nil {
+		return nil, err
+	}
+	if strings.HasPrefix(u.Path, "/public/build/") {
+		return &sourceMapLocation{
+			dir:      setting.StaticRootPath,
```

Comment:
Can you pass in a `setting.Cfg` parameter, and avoid using this global variable? We are doing our very best to get rid of global variables in the backend.

Not sure how to get `setting.Cfg` passed into your code, but that's how it should be done.

Reply:
- [ ] get `setting.Cfg` in here

Reply:
I was thinking that instead of this static location resolution here it would be better to somehow co-opt the existing static file resolution that is used for the `public/*` endpoitns. But could not figure out a way that does not require some extensive refac...

Reply:
Dunno.